uint8 id
uint32 velocityLimit
---
int32 result
